client/client.go

```diff
@@ -199,14 +211,15 @@ func (s3Manager S3Manager) GetBucketRegion(ctx context.Context, bucket string, o
 	return manager.GetBucketRegion(ctx, s3Manager.s3Client, bucket, optFns...)
 }
 
-func NewAwsClient(logger hclog.Logger, accounts []Account, regions []string) Client {
+func NewAwsClient(logger hclog.Logger, accounts []Account, regions, namespaces []string) Client {
 	return Client{
 		ServicesManager: ServicesManager{
 			services: ServicesAccountRegionMap{},
 		},
-		logger:   logger,
-		accounts: accounts,
-		regions:  regions,
+		logger:     logger,
+		accounts:   accounts,
+		regions:    regions,
+		namespaces: namespaces,
```

Comment:
don't see a need for the namespaces string, you can hardcode them in the multiplexer, and just use the WithAccountRegionNamespaces -> we should break it to WithAccount().WithRegion().WithNamespace(), maybe in a different PR.

Reply:
hardcoding led to failing tests because the multiplexer was multiplexing too much, maybe I missed something